src/internet_identity/internet_identity.did

```diff
@@ -140,14 +140,31 @@ type InternetIdentityStats = record {
     canister_creation_cycles_cost: nat64;
 };
 
+// Configuration parameters related to the archive.
+type ArchiveConfig = record {
+    // The allowed module hash of the archive canister.
+    module_hash : blob;
+    // Buffered archive entries limit. If reached, II will stop accepting new anchor operations
+    // until the buffered operations are acknowledged by the archive.
+    // Currently unused: in preparation of switching the archive integration to pull.
+    // Configured value will be discarded for storage_layout_version < 6.
+    entries_buffer_limit: nat64;
+    // The maximum number of entries to be transferred to the archive per call.
+    // Currently unused: in preparation of switching the archive integration to pull.
+    // Configured value will be discarded for storage_layout_version < 6.
+    entries_fetch_limit: nat16;
+    // Polling interval to fetch new entries from II (in nanoseconds).
+    // Currently unused: in preparation of switching the archive integration to pull.
+    // Configured value will be discarded for storage_layout_version < 6.
+    polling_interval_ns: nat64;
```

Comment:
What value do you have in mind here? nanoseconds seems very precise

Reply:
I would suggest using an interval of 1 minute to start with.
I chose the `ns` because it is the general time resolution on the IC.